### PR TITLE
Update tornado version

### PR DESCRIPTION
**Pull request by da5nsy**
Addresses https://github.com/the-turing-way/the-turing-way/security/dependabot/15

### CODE REVIEW

File: book/website/requirements.in [hunk 1]
```diff
@@ -7,3 +7,4 @@ matplotlib
 numpy
 pandas
 sphinx-panels
+tornado>=6.4.2 #temporary fix to https://github.com/the-turing-way/the-turing-way/security/dependabot/15
```
Comment (bsipocz) [inline book/website/requirements.in:10]: I don't think we need the comment, this PR (or the commit message) will have the required metadata for the reason for the pin

**Review by sgibson91 — APPROVED** [review]

**Review by bsipocz — COMMENTED** [review]
I think you need to drop the comment. 

However, I wonder if it's you who need to update the requirements lock file, too or one of our machinery makes it automatically?

(We really need to rename that though to reflect that it's a lock file and not an actual requirements file)

File: book/website/requirements.in (hunk 1 above)
Comment (da5nsy) [inline book/website/requirements.in:10]:
```suggestion
tornado>=6.4.2
```

Comment (da5nsy) [reply book/website/requirements.in]: comment removed

**Comment by da5nsy** [comment]
I got this wrong - I should've updated the `requirements.txt` file rather than the `requirements.in` file, since the builds actually build from `requirements.txt` currently. Tagging https://github.com/the-turing-way/the-turing-way/issues/3862 as related to my confusion 😅 

Switching to https://github.com/the-turing-way/the-turing-way/pull/4024

**Comment by bsipocz** [comment]
No, I think we indeed needed to modify the `in` file, as that is the one tracking the actual dependencies. But then we generate the `txt` file, which is very very confusingly is in fact a lock file for the build rather than actual requirements.

**Comment by JimMadge** [comment]
Yep, _both_ will need to change, and the build process will use the dependencies in `requirements.txt`.

The way to do it is update `requirements.in` and run `pip-compile` to generate `requirements.txt`.

**Pull request merged**